test/e2e/scale.go

```diff
@@ -53,6 +53,10 @@ const (
 	scaleControlPlaneMachineCount = "CAPI_SCALE_CONTROL_PLANE_MACHINE_COUNT"
 	scaleWorkerMachineCount       = "CAPI_SCALE_WORKER_MACHINE_COUNT"
 	scaleMachineDeploymentCount   = "CAPI_SCALE_MACHINE_DEPLOYMENT_COUNT"
+
+	// Note: Names must consist of lower case alphanumeric characters or '-'.
+	scaleClusterNamePlaceholder = "scale-cluster-name-placeholder"
+	scaleNamespacePlaceholder   = "scale-namespace-placeholder"
```

Comment:
nit
```suggestion
	scaleClusterNamespacePlaceholder   = "scale-cluster-namespace-placeholder"
```
for consistency with scaleClusterNamePlaceholder

Reply:
I had it like this because this namespace is also for ClusterClass. But I'll change it. Let's just say this one is for cluster and ClusterClass is then just a logical consequence